src/coreclr/jit/compiler.h

```diff
@@ -9079,13 +9080,22 @@ class Compiler
         return isSIMDClass(clsHnd) || isHWSIMDClass(clsHnd);
     }
 
+    var_types getBaseTypeForPrimitiveNumericClass(CORINFO_CLASS_HANDLE cls);
+
     // Get the base (element) type and size in bytes for a SIMD type. Returns CORINFO_TYPE_UNDEF
     // if it is not a SIMD type or is an unsupported base JIT type.
-    CorInfoType getBaseJitTypeAndSizeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd, unsigned* sizeBytes = nullptr);
+    var_types getBaseTypeAndSizeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd, unsigned* sizeBytes = nullptr);
+
+    var_types getBaseTypeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd)
+    {
+        return getBaseTypeAndSizeOfSIMDType(typeHnd, nullptr);
+    }
+
+    CorInfoType getCorBaseTypeAndSizeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd, unsigned* sizeBytes = nullptr);
 
-    CorInfoType getBaseJitTypeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd)
+    CorInfoType getCorBaseTypeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd)
```

Comment:
Why is this one renamed?

Also why isn't this one using the `var_types` now instead of `CorInfoType`?

Reply:
For refactoring purposes, I left created a `getCorBaseTypeOfSIMDType` for instances where we still need to use the `CorInfoType`. This is mostly for the `gtAuxillaryType` and associated functions.

If we decide that that should also be refactored to use `var_types`, then I believe this family of `Cor` methods can go away.